src/ray/core_worker/core_worker.cc

```diff
@@ -1042,6 +1042,8 @@ void CoreWorker::Shutdown() {
     options_.on_worker_shutdown(GetWorkerID());
   }
 
+  // Force task state events push before exiting the worker.
+  task_event_buffer_->FlushEvents(/* forced= */ true);
```

Comment:
make it consistent with others
```suggestion
  task_event_buffer_->FlushEvents(/*forced=*/ true);
```